app/views/examples/translated/index.njk

```diff
@@ -32,14 +32,27 @@
 
 {% extends "layout.njk" %}
 
+{%block pageTitle%}Translated examples - GOV.UK Frontend{% endblock %}
+
 {% block beforeContent %}
   {{ govukBackLink({
     "href": "/"
   }) }}
 {% endblock %}
 
 {% block content %}
-  <h1 class="govuk-heading-xl">Translated</h1>
+  <h1 class="govuk-heading-xl">Translated examples</h1>
+
+  {{ govukWarningText({
+    text: "Please do not use these translations directly in your application.",
+    iconFallbackText: "Warning"
+  }) }}
+
+  <p class="govuk-body">
+    These examples only illustrate how our components display translated text.
```

Comment:
Thanks @romaricpascal

I can see pushes going up with different text changes being tried out

Think we need to apply the "two thirds" column layout, maybe keep `govuk-body-l` on the opening text, and go with the `--xl` break like some of the other pages do (compare the two visually?) but I'll take a 2nd opinion

Try this:

```html
<div class="govuk-grid-row">
  <div class="govuk-grid-column-two-thirds">
    <h1 class="govuk-heading-xl">Translated examples</h1>

    <p class="govuk-body-l">
      These examples only illustrate how our components display translated text.
      We do not guarantee that they are accurate.
    </p>

    {{ govukWarningText({
      text: "Please do not use these translations directly in your application.",
      iconFallbackText: "Warning"
    }) }}

    <hr class="govuk-section-break govuk-section-break--xl">
  </div>
</div>
```

☝️ @claireashworth suggested putting the warning first to make sure it's seen, but not sure if that's necessary now we're using `govukWarningText()` so I've swapped back

<img width="769" alt="Screenshot showing HTML example above" src="https://user-images.githubusercontent.com/415517/197509277-45bf18f5-57b0-4f1e-967e-3a62c70d3a86.png">